src/object/obj_internal.h

```diff
@@ -204,4 +215,9 @@ obj_dkey2hash(daos_key_t *dkey)
 			       dkey->iov_len, 5731);
 }
 
+/* obj_class.c */
+int obj_ec_auxi_init(void);
+void obj_ec_auxi_fini();
```

Comment:
(style)  Bad function definition - void obj_ec_auxi_fini() should probably be void obj_ec_auxi_fini(void)